### ISSUE TITLE
[Components and pattern standards] Design components and patterns in use are outdated.  (04.08.2)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#60033]
#### VFS team name
Benefits Team 2

#### VFS product name
Financial Status Report (FSR)

#### VFS feature name
FSR Enhancements

#### Point of Contact/Reviewers
Sara Sterkenburg - @sterkenburgsara - IA

*For more information on how to interpret this ticket, please refer to the [Anatomy of a Staging Review issue ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Anatomy-of-a-Staging-Review-Issue-ticket.2060320997.html) guidance on Platform Website.

---

### Platform Issue
Design components and patterns in use are outdated. 

### Issue Details
The privacy agreement component on the form summary page is out of date. It does not indicate to users that the link to review the privacy statement is external and will open in a new page. This indication should be visual and auditory. 

### VA.gov Experience Standard
[Category Number 04, Issue Number 08](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 3.2.4 AA

### Platform Recommendation
Guidance on this component has been updated. Use the most recent [privacy agreement](https://design.va.gov/storybook/?path=/docs/components-va-privacy-agreement--default) component in Storybook and ensure that you follow behavior guidance from the Design System (a small arrow icon will come after the link, and it will be announced for screen readers).

---

### VFS Guidance

- Close the ticket when the issue has been resolved or validated by your Product Owner
- If your team has additional questions or needs Platform help validating the issue, please comment on the ticket
- Some feedback provided may be out of scope for your iteration of the product, however, Platform's OCTO leadership has stated that all identified issues need to be documented and it is still your responsibility to resolve the issue.
- If you do not believe that this Staging Review issue ticket is the responsibility of your team, comment below providing an explanation and who you believe is responsible. Please tag the Point of Contact/Reviewers. Governance team will research and will follow up.

**Comment by HeatherRienks** [comment]
Closing this ticket; [@josephrlee](https://github.com/josephrlee) created ticket [#60295](https://github.com/department-of-veterans-affairs/va.gov-team/issues/60295) for FE crew to complete the work.

**Comment by HeatherRienks** [comment]
FYI @Tom-Davis
